operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -169,6 +169,12 @@ type CassandraDatacenterSpec struct {
 	AdditionalSeeds []string `json:"additionalSeeds,omitempty"`
 
 	Reaper *ReaperConfig `json:"reaper,omitempty"`
+
+	// Determines how many minutes a pod is allowed to exist after
+	// losing readiness before it will automatically be terminated
+	// and replaced.
+	// Default is 10 minutes. Set to 0 to disable
+	TerminateStuckPodsAfterMinutes *int `json:"terminateStuckPodsAfterMinutes,omitempty"`
```

Comment:
Open to naming suggestions if this is too verbose

Reply:
maybe... `terminationDelayForFailedReadinessPods` and we'll just document that the unit is minutes?